Antidote/ActiveCallViewController.h

```diff
@@ -64,11 +65,26 @@
 @property (assign, nonatomic) BOOL speakerSelected;
 
 /**
+ * Set video button selected.
+ */
+@property (assign, nonatomic) BOOL videoButtonSelected;
+
+/**
  * Set whether or not the resume button is hidden.
  */
 @property (assign, nonatomic) BOOL resumeButtonHidden;
 
 /**
+ * Video view of friend
+ */
+@property (strong, nonatomic) UIView *videoView;
+
+/**
+ * Preview layer of video sending
+ */
+@property (strong, nonatomic) CALayer *previewLayer;
```

Comment:
In CallsManager I actually do read the properties from the controller. I had it originally methods like the above, but then CallsManager was continuously invoking methods from `OCTVideoEngine` which overloaded the thread. Do you know if there's a better way?

```
        if (call.friendSendingVideo && ! activeVC.videoView) {
            activeVC.videoView = [self.manager videoFeed];
        }

        if (! call.friendSendingVideo && activeVC.videoView) {
            activeVC.videoView = nil;
        }

        if (! call.videoIsEnabled && activeVC.previewLayer) {
            activeVC.previewLayer = nil;
            activeVC.videoButtonSelected = NO;
        }
```